utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -240,6 +245,30 @@ class HtmlParser private constructor(
       )
     }
 
+    /**
+     * Returns a new [HtmlParser] with the empty entity type and ID for loading images,
+     * doesn't require GCS properties and imageCenterAlign set to false
+     * optionally specified [CustomOppiaTagActionListener] for handling custom Oppia tag events.
+     */
+    fun create(
```

Comment:
I think as per the calling method, only two arguments are used, and these are `displayLocale` and `imageCenterAlign`, so this new factory method should only have those two.  `imageCenterAlign` can be set with a default of `false` here (this parert is already correct). I am not certain about the need for `customOppiaTagActionListener:`

Reply:
Yeah I will also take a look at the test. For customOppiaTagActionListener, I remove it in the input arg. And just to confirm I should set it as null in the function body?

Reply:
Yes, unless you can find any usages where it is not null

Reply:
I updated the tests related to the MathExpressionParserViewModel in the pr description. I search MathExpressionParserViewModel usages in test classes, and run the classes test for them. Let me know if there is any concern with this approach.